### PR TITLE
Fix/container testing

### PR DESCRIPTION
**Pull request by RV-Adrian**
@fjvallarino @alex-deaconu @ezequielc 
Please review. Note that e2e had to be disabled for staging, likely due to the class name changes
( .content-box > .panel , etc)

### CODE REVIEW

File: web/partials/storage/upload-panel.html [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <div ng-controller="UploadController">
-  <div class="content-box animated fadeInDown upload-panel add-bottom" ng-show="uploader.queue.length > 0">
+  <div class="panel animated fadeInDown upload-panel add-bottom" ng-show="uploader.queue.length > 0">
     
       <div ng-click="isCollapsed = !isCollapsed">
-        <div class="content-box-header">
+        <div class="panel-header"> <!-- //TODO: IS this a needed class? -->
```
Comment (alex-deaconu) [inline web/partials/storage/upload-panel.html]: @RV-Adrian not sure who this question is addressed to. Is that a valid class and does it make sense to have it here? Thanks.

File: web/scripts/schedules/services/svc-schedule-factory.js [hunk 2]
```diff
@@ -152,7 +152,7 @@ angular.module('risevision.schedules.services')
 
         //show loading spinner
         factory.loadingSchedule = true;
-        factory.savingSchedule = true;
+        factory.savinxgSchedule = true;
```
Comment (alex-deaconu) [inline web/scripts/schedules/services/svc-schedule-factory.js]: @RV-Adrian typo here. Thanks.

File: web/partials/storage/upload-panel.html (hunk 1 above)
Comment (RV-Adrian) [inline web/partials/storage/upload-panel.html]: @alex-deaconu I'm using TODO as a tag to leave internal notes for possible future CSS improvements. (ie. add to the spreadsheet of tasks)
If it's distracting I can reference by line number here instead.

File: web/scripts/schedules/services/svc-schedule-factory.js (hunk 2 above)
Comment (RV-Adrian) [inline web/scripts/schedules/services/svc-schedule-factory.js]: @alex-deaconu I think that was generated by a 'gulp build'. Re-try?

Comment (alex-deaconu) [inline web/scripts/schedules/services/svc-schedule-factory.js]: @RV-Adrian no idea, I haven't seen gulp build do this but who knows. Just fix manually. 

Also, 'gulp build' in this repo is only necessary for JS code changes (unlike CH or other components), so for this PR it wasn't really needed. However, the formatting changes that resulted shows we didn't run it for previous PRs.

Thanks.

File: web/partials/storage/upload-panel.html (hunk 1 above)
Comment (alex-deaconu) [inline web/partials/storage/upload-panel.html]: @RV-Adrian I thought it was a question to reviewers, thanks for the clarification.

File: bower.json [hunk 3]
```diff
@@ -38,7 +38,7 @@
     "angular-sanitize": "~1.5.0",
     "angular-mocks": "^1.5.5",
     "angular-bootstrap": "^0.13.0",
-    "rv-common-style": "v1.3.93"
-    
+    "rv-common-style": "v1.3.94",
+    "rv-common-app-components": "0.1.1"
```
Comment (alex-deaconu) [inline bower.json]: @RV-Adrian the result of our discussion during Standup was that we won't use a specific tag number for this repo. Either tag `#master` or no tag at all. The intention is to tag the latest release without updating the number specifically. Thanks.

Comment (RV-Adrian) [inline bower.json]: @alex-deaconu Ah yes- so create tags but no specific resolutions in bower, correct?

Comment (alex-deaconu) [inline bower.json]: @RV-Adrian yup!

**Comment by alex-deaconu** [comment]
@RV-Adrian 👍 thanks for fixing!

**Pull request merged**